src/components/Network.js

```diff
@@ -11,6 +11,19 @@ import ReactLoading from 'react-loading';
 import dateFormatter from '../utils/date';
 import colors from '../index.scss';
 
+const SyncStates = {
+  IN_SYNC: 'in-sync',
+  BEHIND: 'behind',
+  AHEAD: 'ahead',
+  UNKNOWN: 'unknown',
+};
+
+const SyncStatesDescription = {
+  [SyncStates.IN_SYNC]: 'Synchronized',
+  [SyncStates.BEHIND]: <span title="Behind of us">Not synchronized <i class="fa fa-question-circle"></i></span>,
```

Comment:
I'd change it to "Synchronizing..." for both of them.

Reply:
Done